MIGRATING.md

```diff
@@ -38,6 +93,26 @@ major releases of `cosmwasm`. Note that you can also view the
   ctx.add_attribute("destination", &to_addr);
   ```
 
+- Add result type to `Bucket::update` and `Singleton::update`:
+
+  ```rust
+  // before
+  bucket.update(b"maria", |mayd: Option<Data>| {
+    let mut d = mayd.ok_or(StdError::not_found("Data"))?;
+    old_age = d.age;
+    d.age += 1;
+    Ok(d)
+  })
+
+  // after
+  bucket.update(b"maria", |mayd: Option<Data>| -> StdResult<_> {
```

Comment:
Wow, even here we allow any type. You were thorough

Reply:
Jap, some contracts required the ability to use their custom type in update